### PR TITLE
Handle string 0 in math helper

### PR DESCRIPTION
**Pull request by sethkinast**
Unbitrotted #54

### CODE REVIEW

File: lib/dust-helpers.js [hunk 1]
```diff
@@ -270,42 +270,45 @@ var helpers = {
               return null;
           };
 
-      key  = dust.helpers.tap(key, chunk, context);
-      operand = dust.helpers.tap(operand, chunk, context);
+      key = parseFloat(dust.helpers.tap(key, chunk, context));
+      operand = parseFloat(dust.helpers.tap(operand, chunk, context));
```
Comment (smfoote) [inline lib/dust-helpers.js]: Do you need to do any error checking here?

**Comment by smfoote** [comment]
Looks good

File: lib/dust-helpers.js (hunk 1 above)
Comment (sethkinast) [inline lib/dust-helpers.js]: Shouldn't need to. Anything icky coerces to `NaN` and that's the eventual math output.

**Comment by sethkinast** [comment]
I promise to add tests, but the framework inside -helpers doesn't support testing log messages.

I'll port that part over from main, but I don't want to do it as part of this PR.

Tests already exist for div by 0 and such-- the only test that doesn't exist is that the correct message is logged.

Is OK?

**Comment by smfoote** [comment]
Works for me.

**Pull request merged**